src/world/Actor/Chara.cpp

```diff
@@ -355,13 +355,16 @@ bool Sapphire::Entity::Chara::checkAction()
 
 void Sapphire::Entity::Chara::update( uint64_t tickCount )
 {
-  updateStatusEffects();
-
-  if( std::difftime( static_cast< time_t >( tickCount ), m_lastTickTime ) > 3000 )
+  if( isAlive() )
```

Comment:
this will work for intended purposes, but some status effects still 'tick' even if the actor is dead (eg. ifrit searing wind)

Reply:
Yeah, I updated it to only stop dot/hot.